pype/plugins/blender/load/load_model.py

```diff
@@ -162,16 +163,24 @@ def update(self, container: Dict, representation: Dict):
         assert libpath.is_file(), (
             f"The file doesn't exist: {libpath}"
         )
-        assert extension in pype.hosts.blender.plugin.VALID_EXTENSIONS, (
+        assert extension in plugin.VALID_EXTENSIONS, (
             f"Unsupported file: {libpath}"
         )
 
         collection_metadata = collection.get(
             blender.pipeline.AVALON_PROPERTY)
         collection_libpath = collection_metadata["libpath"]
-        objects = collection_metadata["objects"]
         lib_container = collection_metadata["lib_container"]
 
+        obj_container = [
```

Comment:
Advice for future PR:
1.) I see you use these lines at multiple places maybe you can create function which do obtaining container by it's name.
2.) You always go through all collections even if you've already found what you need, sometimes more lines of code can be more efficient.
```
check_name = collection_metadata["obj_container"].name

obj_container = None
for col in bpy.data.collections:
    if col.name == check_name and col.library is None:
        obj_container  = col
        # stop iteration
        break
```

Reply:
You are absolutely right! This is the rush to make it work! I'll change those, thanks!